### PR TITLE
Do not need to iterate values

### PR DESCRIPTION
**Pull request by bryancunningham-okta**
[REPLACE_WITH_JIRA_TICKET_NUMBER](https://oktainc.atlassian.net/browse/REPLACE_WITH_JIRA_TICKET_NUMBER)

## Summary
- Fixes a bug where I was trying to iterate an undefined value

## Testing & Screenshots

- [ ] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

File: packages/odyssey-react-mui/src/labs/DataFilters.tsx [hunk 1]
```diff
@@ -604,6 +604,7 @@ const DataFilters = ({
                               <AutocompleteInnerContainer>
                                 <Autocomplete
                                   hasMultipleChoices
+                                  isCustomValueAllowed
```
Comment (bryancunningham-okta) [inline packages/odyssey-react-mui/src/labs/DataFilters.tsx:607]: @KevinGhadyani-Okta Do you think we want to just allow this all the time within data filters?

File: packages/odyssey-react-mui/src/labs/DataFilters.tsx [hunk 2]
```diff
@@ -535,13 +535,13 @@ const DataFilters = ({
...
-
-    return undefined;
-  }, [filterPopoverCurrentFilter]);
+  const autoCompleteValue = useMemo(
+    () =>
+      filterPopoverCurrentFilter?.id
+        ? (inputValues[filterPopoverCurrentFilter.id] as Option[])
+        : undefined,
+    [filterPopoverCurrentFilter, inputValues],
+  );
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/labs/DataFilters.tsx:544]: Fine with this change.

File: packages/odyssey-react-mui/src/labs/DataFilters.tsx (hunk 1 above)
Comment (KevinGhadyani-Okta) [reply packages/odyssey-react-mui/src/labs/DataFilters.tsx:607]: I want more information on what this does.

**Review by KevinGhadyani-Okta — DISMISSED** [review]

File: packages/odyssey-react-mui/src/labs/DataFilters.tsx (hunk 1 above)
Comment (bryancunningham-okta) [reply packages/odyssey-react-mui/src/labs/DataFilters.tsx:607]: For posterity, we chatted about this on Slack but if anyone else comes across this, this prop allows the user to enter any value into the autocomplete. They are not required to select an option from the drop down.

**Pull request closed without merging**